flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ArchivedExecutionGraph.java

```diff
@@ -337,7 +337,7 @@ public static ArchivedExecutionGraph createFrom(
      * Create a sparse ArchivedExecutionGraph for a job while it is still initializing. Most fields
      * will be empty, only job status and error-related fields are set.
      */
-    public static ArchivedExecutionGraph createFromInitializingJob(
+    public static ArchivedExecutionGraph createSparseArchivedExecutionGraph(
```

Comment:
The renaming became necessary because it would be used in different places (in initialization and cleanup phase for the job). Hence, the old name was wrong. 

I try to keep the commits separated logically in such big PRs to make it possible to go over the PR commit by commit. Is this something that helps or is it unnecessary effort?